### PR TITLE
Remove the pre-2002 data from the NC data import

### PR DESCRIPTION
**Pull request by jbradberry**
According to the client, the various NC police agencies were not
consistently reporting the data until 1 Jan 2002, so any data before
that will be a biased sample.  The NC Highway Patrol, however, was
consistently reporting, so we leave those data points in the data set.

### CODE REVIEW

File: nc/data/importer.py [hunk 1]
```diff
@@ -278,3 +281,15 @@ def copy_from(destination, nc_csv_path):
     if settings.DATABASE_ETL_USER:
         cmd.append(settings.DATABASE_ETL_USER)
     call(cmd)
+
+    # Remove all stops and related objects that are before 1 Jan 2002, when everyone
+    # started reporting consistently.  Don't clear out the NC State Highway Petrol
```
Comment (dchukhin) [inline nc/data/importer.py]: "Petrol" 🛢

**Comment by dchukhin** [comment]
https://caktus.atlassian.net/browse/TRH-2728

This pull request now includes:
 - Jeff’s backend changes to remove pre-2002 data, except for the NC Highway Patrol data
 - front-end changes to show all data from all years in the charts
 - front-end changes to show display years between 2000 and this year in the traffic stop search for all agencies

**Comment by kialam** [comment]
When I see the data chart for agency 281, I expect to not see prior to 2002 data but instead got:

screen shot 2018-02-01 at 2 04 21 pm

**Comment by kialam** [comment]
♻️

**Pull request merged**